@navikt/ds-css/form/error-summary.css

```diff
@@ -0,0 +1,27 @@
+.navds-error-summary {
+  display: block;
+  background-color: #fff;
+  padding: var(--navds-spacing-4);
+  border-radius: 0;
```

Comment:
Dette er også default. Er det noen grunn til å ha denne linja?

Reply:
Var med fra Feiloppsummering. Ser ingen grunn for den